src/main/java/account/DepositAccount.java

```diff
@@ -0,0 +1,84 @@
+package account;
+
+import interest.DepositInterest;
+import interest.InterestCalculator;
+import exception.account.InsufficienBalancetException;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.math.BigDecimal;
+
+@Getter
+@Setter
+public class DepositAccount implements Account{
+
+    private static final String DEPOSIT_COMPLETE = "입금완료! [잔고] = %s";
+    private static final String WITHDRAW_COMPLETE = "출금완료! [잔고] = %s";
+    private static final String INSUFFICIENT_BALANCE = "잔액부족! [잔고] = %s 처음부터 다시 입력해주세요";
+    private static final String REMITTANCE_COMPLETE = "송금완료! [잔고] = %s";
+    private static final String ACCOUNT_INACTIVE = "[계좌상태] = 비활성화";
+    private static final String ACCOUNT_ACTIVE = "[계좌상태] = 활성화";
+    private static final String DISPLAY_ACCOUNT_INFO = "[계좌유형] = %s [계좌번호] = %s [소유자] = %s [잔액] = %s [활성화 여부] = %s";
+
+    InterestCalculator interestCalculator = new DepositInterest();
```

Comment:
```suggestion
    private final InterestCalculator interestCalculator = new DepositInterest();
```
접근 제어자를 필요에 따라 알맞게 설정해 줘야할 것 같습니다.